devnet-sdk/system/system.go

```diff
@@ -45,11 +45,13 @@ func (s *system) Identifier() string {
 func (s *system) addChains(chains ...*descriptors.Chain) error {
 	for _, chainDesc := range chains {
 		if chainDesc.ID == "" {
-			l1, err := chainFromDescriptor(chainDesc)
-			if err != nil {
-				return fmt.Errorf("failed to add L1 chain: %w", err)
+			if chainDesc.Name == "Ethereum" {
+				l1, err := chainFromDescriptor(chainDesc)
+				if err != nil {
+					return fmt.Errorf("failed to add L1 chain: %w", err)
+				}
+				s.l1 = l1
 			}
```

Comment:
when L1 is not set becaue its name is not `Ethereum`, I think its helpful to log or throw an error in this case

Reply:
Let's leave that for later PR, I have a PR coming that will set the L1 chain ID that I think this belongs in better.